source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,116 @@
+.. _pymongo-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+The Atlas Search feature enables you to perform full-text searches on
+collections hosted on MongoDB Atlas. The indexes specify the behavior of
+the search and which fields to index.
+
+To learn more about MongoDB Atlas Search, see the
+:atlas:`Atlas Search Indexes </atlas-search/atlas-search-overview/#fts-indexes>`
+documentation.
+
+You can call the following methods on a collection to manage your Atlas Search
+indexes:
+
+- ``create_search_index()``
+- ``create_search_indexes()``
+- ``list_search_indexes()``
+- ``update_search_index()``
+- ``drop_search_index()``
+
+.. note::
+
+   The Atlas Search Index management methods run asynchronously. The
+   driver methods can return before confirming that they ran
+   successfully. To determine the current status of the indexes, call the
+   ``list_search_indexes()`` method.
+
```

Comment:
This page doesn't mention the sample collection; is that needed here?

Reply:
I don't think any of the code examples for this will be using the sample collection so I left it off of this page